content/bn/docs/reference/glossary/docker.md

```diff
@@ -0,0 +1,17 @@
+---
+title: ডকার
+id: docker
+date: 2018-04-12
+full_link: https://docs.docker.com/engine/
+short_description: >
+  ডকার হলো একটি সফ্টওয়্যার প্রযুক্তি যা অপারেটিং-সিস্টেম-লেভেল ভার্চুয়ালাইজেশন প্রদান করে যা কন্টেইনার নামেও পরিচিত।
+
+aka:
+tags:
+- fundamental
+---
+ডকার (বিশেষ করে, ডকার ইঞ্জিন) হলো একটি সফ্টওয়্যার প্রযুক্তি যা অপারেটিং-সিস্টেম-লেভেল ভার্চুয়ালাইজেশন প্রদান করে যা {{< glossary_tooltip text="কন্টেইনার" term_id="container" >}} নামেও পরিচিত।
+
+<!--more-->
+
+ডকার লিনাক্স কার্নেলের রিসোর্স আইসোলেশন বৈশিষ্ট্যগুলি ব্যবহার করে যেমন cgroups এবং কার্নেল নেমস্পেস, এবং একটি ইউনিয়ন-সক্ষম ফাইল সিস্টেম যেমন OverlayFS এবং অন্যান্যগুলিকে স্বাধীন কন্টেইনারগুলিকে একটি একক লিনাক্স ইনস্ট্যান্সের মধ্যে চালানোর অনুমতি দেওয়ার জন্য, ভার্চুয়াল মেশিনগুলি (VMs) শুরু করা এবং রক্ষণাবেক্ষণের ওভারহেড এড়ানো।
```

Comment:
Hello @SAShameem , I think it will be ideal to use `ফিচারগুলি` instead of `বৈশিষ্ট্যগুলি` for `features`.

Reply:
OK. I am changing  it